### ISSUE TITLE
No implementation of MHIM for DSMIL

### ISSUE THREAD

**Issue by mayonezu216** [DearCaat/MHIM-MIL#3]
Hi, in the paper, MHIM(DSMIL) shows very good performance in many experiments. However, I can only find TransMIL and ABMIL models with MHIM. Could you provide the code of DSMIL with MHIM, the hyperparameters for masks and pretrained DSMIL models?

**Comment by DearCaat** [comment]
Thanks for attention. I have updated the imple of DSMIL, pls kindly check it: 
_Readme:_ https://github.com/DearCaat/MHIM-MIL#dsmil-same-hyper-param-with-abmil-maybe-not-optimal,
_Code:_ https://github.com/DearCaat/MHIM-MIL/blob/master/modules/datten.py#L167
As for the hyper-param about DSMIL, I only try the same hyper-param with AB-MIL. I'm not sure it's the optimal, but it performs well on the C16 and TCGA-NSCLC.

**Comment by mayonezu216** [comment]
Thanks very much! That is useful.
